geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -531,21 +532,12 @@ protected void runDispatcher() {
 
   private boolean handleAuthenticationExpiredException(AuthenticationExpiredException expired)
       throws InterruptedException {
-    long reAuthenticateWaitTime =
-        getSystemProperty(RE_AUTHENTICATE_WAIT_TIME, DEFAULT_RE_AUTHENTICATE_WAIT_TIME);
-    // for old client, don't wait for re-auth but unregister this proxy completely.
-    if (getProxy().getVersion().isOlderThan(RE_AUTHENTICATION_START_VERSION)) {
-      synchronized (_stopDispatchingLock) {
-        String message =
-            "Authentication expired for a client with a version less than Geode 1.15. Cannot re-authenticate an older client "
-                + " that has a server to client queue for CQs or interest registrations. "
-                + "Please upgrade your client to Geode 1.15 or greater to allow re-authentication.";
-        logger.warn(message);
-        pauseOrUnregisterProxy(expired);
-      }
+    if (unRegisterUnsupportedClient(expired)) {
```

Comment:
Unregister is one word, so the method name following naming standards should not camelCase the `unRegister`.